install-instructions.php

```diff
@@ -359,7 +359,16 @@
 		      <a href="<?php echo $DOWNLOAD_CLIENT_DESKTOP_TEST_LINUX; ?>" class="btn btn-lg btn-default"><i class="icon-linux"></i> Linux</a>
 		      <a href="<?php echo $DOWNLOAD_CLIENT_DESKTOP_TEST_SOURCES; ?>" class="btn btn-lg btn-default"><i class="icon-archive"></i> Sources</a>
 		    </div>
-		    <?php } ?>
+		    <?php } else {?>
+			    <h4>Desktop Clients <small><?php echo $VERSIONS_CLIENT_DESKTOP_TESTING; ?></small></h4>
+	 		    <p>Our desktop team provides regular nightly builds. Please report any issues to the <a href="https://github.com/owncloud/mirall/issues" target="_blank">tracker</a>.</p>
+			    <div class="btn-group">
+			      <a href="http://download.owncloud.com/desktop/daily/" class="btn btn-lg btn-default"><i class="icon-windows"></i>  Windows</a>
+			      <a href="http://download.owncloud.com/desktop/daily/>" class="btn btn-lg btn-default"><i class="icon-apple"></i> Mac</a>
+			      <a href="https://software.opensuse.org/download.html?project=isv%3AownCloud%3Acommunity%3Anightly&package=owncloud-client" class="btn btn-lg btn-default"><i class="icon-linux"></i> Linux</a>
+			      <a href="https://github.com/owncloud/client" class="btn btn-lg btn-default"><i class="icon-archive"></i> Sources</a>
+			    </div>
+			 <?php } ?>
 		  </div>
```

Comment:
Weird, are you sure? Cache-thing in WP flushed?
https://owncloud.org/install/#testing-development